exporters/sql_export/lc_for_jobs_db.py

```diff
@@ -0,0 +1,80 @@
+""" This program makes an actual state sqlite-database for use by
+jobs run under job-runner as opposed to the actual state used
+by customers. It is meant to be run just after the nightly imports
+to be used as a speed up in comparison with hitting MO's rest interface.
+"""
+import click
+import json
+import logging
+import pathlib
+
+
+from exporters.sql_export.sql_export import SqlExport
+from sqlalchemy import create_engine
+
+
+LOG_LEVEL = logging.DEBUG
+LOG_FILE = "lc-for-jobs.log"
+
+logger = logging.getLogger("lc-for-jobs")
+
+
+def get_engine(dbpath=None):
+    if dbpath is None:
+        cfg_file = pathlib.Path.cwd() / "settings" / "settings.json"
+        if not cfg_file.is_file():
+            raise Exception("No setting file")
+        settings = json.loads(cfg_file.read_text())
+        dbpath = settings["lc-for-jobs.actual_db_name"]
+
+    db_string = "sqlite:///{}.db".format(dbpath)
+    return create_engine(db_string)
+
+
+@click.group()
+def cli():
+    # Solely used for command grouping
+    pass
+
+
+@cli.command()
+@click.option("--resolve-dar/--no-resolve-dar", default=False)
+def sql_export(resolve_dar):
+
+    # Load settings file
+    cfg_file = pathlib.Path.cwd() / "settings" / "settings.json"
+    if not cfg_file.is_file():
+        raise Exception("No setting file")
```

Comment:
Overvej caching load of settings / lazy load, i stedet for at loade både her og i `get_engine`

Reply:
Overvej settings load aka hvordan det er lavet med MED util tingen

Reply:
Settings load er det same mange steder. At bruge 3 linjer på at loade den er fint. Koden er gennemskuelig. 
Alle programmer afvikles fra roden. 
Nøje Overvejet.